apps/builder/app/builder/features/style-panel/sections/backgrounds/background-layers.ts

```diff
@@ -150,6 +161,11 @@ const getLayersValue = (styleValue?: StyleValueInfo) => {
   if (clonedStyleValue?.local?.type === "layers") {
     return clonedStyleValue.local;
   }
+
+  if (clonedStyleValue?.previousSource?.value?.type === "layers") {
+    return clonedStyleValue?.previousSource?.value;
+  }
```

Comment:
Might need also `nextSource` I added yesterday. It's same remote style source but shows alert icon and says the value is overridden